Makefile

```diff
@@ -19,6 +19,7 @@ lint:
 	$(GOLINT) $$($(GOLIST_PKG))
 format:
 	find . -name '*.go' | xargs gofmt -s -w
+	find . -name '*.go' | xargs goimports -w
```

Comment:
I consider to add the code to add goimports if there is no goimports and maybe we can delete `find . -name '*.go' | xargs gofmt -s -w`.

Reply:
I think `goimports` is not a standard package (It's kind of de facto standard) . And test job in screwdriver.yaml does not have a running `goimports` step. There are differences.

Reply:
thx
fixed